corehq/apps/data_interfaces/interfaces.py

```diff
@@ -85,12 +87,22 @@ def rows(self):
     @property
     def report_context(self):
         context = super(CaseReassignmentInterface, self).report_context
-        active_users = self.get_all_users_by_domain(user_filter=tuple(HQUserType.all()), simplified=True)
+        if not self.request.can_access_all_locations:
+            accessible_location_ids = (SQLLocation.active_objects.accessible_location_ids(
+                self.request.domain,
+                self.request.couch_user)
+            )
+            accessible_user_ids = user_ids_at_locations(accessible_location_ids)
+            active_users = self.get_all_users_by_domain(user_filter=tuple(HQUserType.all()), simplified=True,
+                                                        user_ids=tuple(accessible_user_ids))
```

Comment:
Non-blocking, but I'm concerned about the performance of this call.  I just looked at the definition of `get_all_users_by_domain`, and it does `CommCareUser.get` for every id.  That means potentially hundreds or thousands of database hits.  It would be faster to do this in bulk.  Something like this should do it:

```python
user_query = UserES().domain(domain).location(accessible_location_ids)
active_users = get_simplified_users(user_query)
```

Or some other way of calling `_report_user_dict`.  (I just ran in to something similar recently and added `get_simplified_users` to make it easy to go from ES queries to report users)